README.md

```diff
@@ -40,6 +40,147 @@ If you'd like to check out the code and contribute, [join us on github](https://
+
+``` php
+define('LIVEBLOG_FEATURES', 'commands, hashtags, emojis, authors');
+```
+
+The command system has one inbuilt command:
+
+`/key`: Which defines an entry to a key event, it adds the meta key to entry `liveblog_key_entry`. In your theme you can add the shortcode `[liveblog_key_events]` where entries used with the key command will be inserted - it acts an anchor system to jump to parts of the blog. If the user has enabled HTML5 notification and the window is not currently in focus they will receive a notification about that entry.
+
+You can add new commands easily with a filter, the most basic command will add a class to entry, so you could do a simple  `/highlight` which would add `type-highlight` to the entry container letting you style a new background color:
+
+``` php
+add_filter( 'liveblog_active_commands',  array( __CLASS__, 'add_highlight_command' ), 10 );
+
+
+public static function add_highlight_command( $commands ) {
+  $commands[] = highlight;
+  return $commands;
+}
+```
+
+A command can have both a filter called before the entry is saved, or an action that is called after it’s saved:
+
+``` php
+apply_filter( "liveblog_command_{$command}_before", $arg );
+do_action( "liveblog_command_{$command}_after", $arg );
+```
+
+#### Customizing Key Events Shortcode
+
+As mentioned earlier you can add the key events section by using `[liveblog_key_events]`. If you wish to change the title from the default `Key Events` then you a title attribute.
+
+ ``` php
+ [liveblog_key_events title="My New Title"]
+ ```
+ If want to remove the title, good example when placing the shortcode in a widget.
+
+ ``` php
+ [liveblog_key_events title="false"]
+ ```
+
+A key event entry can be altered in to two ways:
+
+**Template:** Is how each entry will be rendered, there are two inbuilt templates (list or timeline). You can add your own using a filter:
+
+```php
+add_filter( 'liveblog_key_templates', 'add_template', 'liveblog-key-custom-css-class');
+
+function add_template( $templates ) {
+  $templates['custom'] = array( '{theme}/key-events.php', 'div' );
+  return $templates;
+}
+```
+There's a few things to note here:
+
+* `{theme}` is a shorthand to point to the current active theme directory, in this case we our loading template file `key-events.php`.
+* `div` is where we set the element type the wraps all entries, in the case where you wanted to built a list you would set this to `ul`.
+* `liveblog-key-custom-css-class` is a class that will added the wrapper element of the entry to help with styling, in this case that'd look like: `<div class="liveblog-key-custom-css-class">...</div>`
+
+An example of a template file is:
+
+```php
+<div <?php echo $css_classes; ?> >
+	<a href="#liveblog-entry-<?php echo $entry_id; ?>">
+		<?php echo WPCOM_Liveblog_Entry_Key_Events::get_formatted_content( $content, $post_id ); ?>
+	</a>
+</div>
+```
+
+**Format:** Is how each entries content is filtered, there is three inbuilt formats:
```

Comment:
I've added a little more info to the docs on formats so it makes a bit more sense now:

Formats add an extra level of control to how the key events section looks. If using the `timeline` template with `first sentence` and the following is entered into the new entry box:
```
New iPad announced. With plan to ship next month, pre-orders starting 23rd September retailing at $499 for 16gb $599 32gb. /key
```

The main feed and notification would show the full text, and the key events section would only show:
```
New iPad announced
```